localedata/locales/ca_ES

```diff
@@ -144,7 +150,7 @@ mon     "de gener";/
         "d<U2019>octubre";/
         "de novembre";/
         "de desembre"
-d_t_fmt "%a %d %b %Y %T %Z"
+d_t_fmt "%A, %e %B de %Y, %T %Z"
```

Comment:
Looks good if this is what you want. ```"%d"``` displays a day number with leading zero: ```"01"```, ```"02"```... ```"%e"``` displays a day number with leading space: ```" 1"```, ```" 2"```... If you want no leading zero and no leading space then you should use ```"%-d"``` or ```"%-e"```, both produce the same result.